frontend/styles/components/atoms/Dropdown.module.scss

```
@@ -1,17 +1,20 @@
+@import '../../Variables.module.scss';
+
 .dropdown {
     position: relative;
-    background-color: white;
-    // border: 1px solid #ccc;
+    // No background here -- the button inside already has its own white, rounded fill;
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)